packages/protocol/contracts/bridge/erc721/libs/LibERC721.sol

```diff
@@ -0,0 +1,208 @@
+// SPDX-License-Identifier: MIT
+//  _____     _ _         _         _
+// |_   _|_ _(_) |_____  | |   __ _| |__ ___
+//   | |/ _` | | / / _ \ | |__/ _` | '_ (_-<
+//   |_|\__,_|_|_\_\___/ |____\__,_|_.__/__/
+
+pragma solidity ^0.8.20;
+
+import {AddressResolver} from "../../../common/AddressResolver.sol";
+import {
+    Create2Upgradeable
+} from "@openzeppelin/contracts-upgradeable/utils/Create2Upgradeable.sol";
+import {
+    ERC721Upgradeable
+} from "@openzeppelin/contracts-upgradeable/token/ERC721/ERC721Upgradeable.sol";
+import {NFTVault} from "../../NFTVault.sol";
+import {BridgedERC721} from "../BridgedERC721.sol";
+import {IBridge} from "../../IBridge.sol";
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+
+library LibERC721 {
```

Comment:
Why we have to have a library for ERC721? Otherwise some contracts are too large?